debian/copyright

```diff
@@ -1,59 +1,1130 @@
+ mysys/guess_malloc_library.c
+ mysys/hash.c
+ mysys/lf_alloc-pin.c
+ mysys/lf_dynarray.c
+ mysys/list.c
+ mysys/mf_cache.c
+ mysys/mf_dirname.c
+ mysys/mf_fn_ext.c
+ mysys/mf_format.c
+ mysys/mf_getdate.c
+ mysys/mf_iocache2.c
+ mysys/mf_iocache.c
+ mysys/mf_keycache.c
+ mysys/mf_keycaches.c
+ mysys/mf_loadpath.c
+ mysys/mf_pack.c
+ mysys/mf_path.c
+ mysys/mf_sort.c
+ mysys/mf_tempdir.c
+ mysys/mf_tempfile.c
+ mysys/mf_unixpath.c
+ mysys/my_access.c
+ mysys/my_addr_resolve.c
+ mysys/my_alloc.c
+ mysys/my_atomic_writes.c
+ mysys/my_basename.c
+ mysys/my_bit.c
+ mysys/my_bitmap.c
+ mysys/my_chsize.c
+ mysys/my_compare.c
+ mysys/my_compress.c
+ mysys/my_copy.c
+ mysys/my_cpu.c
+ mysys/my_create.c
+ mysys/my_default.c
+ mysys/my_delete.c
+ mysys/my_dlerror.c
+ mysys/my_error.c
+ mysys/my_file.c
+ mysys/my_fopen.c
+ mysys/my_fstream.c
+ mysys/my_gethwaddr.c
+ mysys/my_getncpus.c
+ mysys/my_getopt.c
+ mysys/my_getsystime.c
+ mysys/my_getwd.c
+ mysys/my_init.c
+ mysys/my_largepage.c
+ mysys/my_lib.c
+ mysys/my_likely.c
+ mysys/my_lock.c
+ mysys/my_lockmem.c
+ mysys/my_malloc.c
+ mysys/my_mess.c
+ mysys/my_mmap.c
+ mysys/my_once.c
+ mysys/my_open.c
+ mysys/my_pread.c
+ mysys/my_pthread.c
+ mysys/my_quick.c
+ mysys/my_rdtsc.c
+ mysys/my_read.c
+ mysys/my_redel.c
+ mysys/my_rename.c
+ mysys/my_rnd.c
+ mysys/my_seek.c
+ mysys/my_sleep.c
+ mysys/my_static.c
+ mysys/my_symlink.c
+ mysys/my_sync.c
+ mysys/my_thr_init.c
+ mysys/my_uuid.c
+ mysys/my_wincond.c
+ mysys/my_winerr.c
+ mysys/my_winfile.c
+ mysys/my_winthread.c
+ mysys/my_wintoken.c
+ mysys/my_write.c
+ mysys/ptr_cmp.c
+ mysys/safemalloc.c
+ mysys_ssl/openssl.c
+ mysys/stacktrace.c
+ mysys/string.c
+ mysys/test_charset.c
+ mysys/testhash.c
+ mysys/test_thr_mutex.c
+ mysys/thr_alarm.c
+ mysys/thr_lock.c
+ mysys/thr_mutex.c
+ mysys/thr_rwlock.c
+ mysys/tree.c
+ mysys/typelib.c
+ mysys/waiting_threads.c
+ mysys/wqueue.c
+ scripts/comp_sql.c
+ sql-common/client.c
+ sql-common/client_plugin.c
+ sql-common/my_time.c
+ sql-common/my_user.c
+ sql/password.c
+ sql/plistsort.c
+ sql/udf_example.c
+ sql/winservice.c
+ strings/conf_to_src.c
+ strings/ctype-big5.c
+ strings/ctype.c
+ strings/ctype-cp932.c
+ strings/ctype-czech.c
+ strings/ctype-euc_kr.c
+ strings/ctype-extra.c
+ strings/ctype-gb2312.c
+ strings/ctype-gbk.c
+ strings/ctype-latin1.c
+ strings/ctype-mb.c
+ strings/ctype-simple.c
+ strings/ctype-sjis.c
+ strings/ctype-uca.c
+ strings/decimal.c
+ strings/do_ctype.c
+ strings/dump_map.c
+ strings/json_lib.c
+ strings/json_normalize.c
+ strings/my_strchr.c
+ strings/my_vsnprintf.c
+ strings/strmake.c
+ strings/uca-dump.c
+ strings/unidata-dump.c
+ strings/utr11-dump.c
+ strings/xml.c
+ tests/bug25714.c
+ tests/mysql_client_fw.c
+ tests/mysql_client_test.c
+ tests/thread_test.c
+ vio/test-ssl.c
+ vio/test-sslclient.c
+ vio/test-sslserver.c
+ vio/vio.c
+ vio/viopipe.c
+ vio/viosocket.c
+ vio/viossl.c
+ vio/viosslfactories.c
+ vio/viotest-ssl.c
+Copyright: 2000, 2010, Oracle and/or its affiliates.
```

Comment:
Can we abbreviate this down to the minimal encompassing set of years/year ranges for each entity?

Reply:
I'll chew them down.. autotool to fetch info gives huge list and seems it does not make `sort -u`